modules/ducktests/tests/ignitetest/services/utils/auth/__init__.py

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains authentication classes and utilities.
+"""
+
+DEFAULT_AUTH_PASSWORD = "ignite"
+DEFAULT_AUTH_USERNAME = "ignite"
+
+
+def get_credentials_from_globals(_globals: dict = None, user: str = 'server'):
+    """
+    Parse globals with structure like that.
```

Comment:
should be explained by proper test, not by such comment

Reply:
test added

Reply:
But we still have a comment, which will be broken on first refactoring.

Reply:
replace with reference to unit test